modules/custom-cert-new-install-roxctl.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: PROCEDURE
+[id="custom-cert-new-install-roxctl_{context}"]
+= Adding a custom certificate during a new installation with roxctl CLI
+
+If you are installing {product-title} using the `roxctl` CLI, provide the certificate and key files when you run the installation program.
+
+.Procedure
+
+* For the non-interactive installer, use the `--default-tls-cert` and   `--default-tls-key` options:
```

Comment:
This is only going to generate the keys. Lets describe where they are located after you generate them.